src/ItemList/ItemList.module.css

```diff
@@ -0,0 +1,50 @@
+.item {
+  position: relative;
+}
+
+.list {
+  font-style: italic;
+}
+
+button.delete {
+  background: url(../img/delete.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  opacity: .3;
+  animation: opacityUnHover ease .5s;
+  position: absolute;
+  right: 10px;
+  top: calc( 50% - 14px);
```

Comment:
Codacy found an issue: [Unexpected whitespace after "(" in a single-line function (function-parentheses-space-inside)](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=5838745)